### PR TITLE
Adds navigation to error line on repl.html #604

### PR DESCRIPTION
**Pull request by abhisheksoni27**
As mentioned in #604

This highlights the erroneous line (when the link is clicked, of course) in the case of an Error.

I have used `red` color to show the link, is that all right?

Here's how it looks: 
prepack

Clicking on the link **highlights** that particular line in the `editor`.

### CODE REVIEW

File: js/repl.js [hunk 1]
```diff
@@ -77,7 +107,7 @@ var input = createEditor(document.querySelector('.input .repl'));
 input.setValue([
   '(function() {',
   '  function fib(x) {',
-  '    return x <= 1 ? x : fib(x - 1) + fib(x - 2);',
+  '    return x <= 1 ? x : fi(x - 1) + fib(x - 2);',
```
Comment (cblappert) [inline js/repl.js]: Please leave this as `fib`

File: js/repl.js [hunk 2]
```diff
@@ -57,9 +58,30 @@ function compile() {
         }
         output.setValue(code, -1);
       } else if (result.type === 'error') {
+        let errorText = result.data;
+        if(!errorText.startsWith('Unexpected')){
+        errorOutput.style.display = 'block';
```
Comment (cblappert) [inline js/repl.js]: Please use style of existing if statements -- spaces around parentheses and indentation of blocks.

File: js/repl.js [hunk 3]
```diff
@@ -57,9 +58,30 @@ function compile() {
...
+        replOutput.style.display = 'none';
+        errorOutput.textContent = errorText;        
+        } else {
+        let errorLineLink = document.createElement('a');
```
Comment (cblappert) [inline js/repl.js]: Fix indentation here too

File: js/repl.js (hunk 1 above)
Comment (abhisheksoni27) [inline js/repl.js]: I changed that for testing. Forgot to change it back. :frowning:

**Comment by abhisheksoni27** [comment]
@cblappert How will we add eslint to the editor? I don't think it's available as a JS file, is it?

**Pull request merged**